install/kubernetes/helm/istio/values.yaml

```diff
@@ -14,6 +14,13 @@ global:
   tag: circleci-nightly
   proxy:
     image: proxy
+    resources:
```

Comment:
This is far too low for a normal cluster. Keep in mind pilot in 0.8 may still handle v1 - which is very resource intentsive.

This setting should be good enough for a default install / medium cluster.

Reply:
I can update the default resources to only have `requests` which is minimum required compute resources.
Alternatively, we can keep empty resources as it was (close this PR). The intention of this PR was to make it easier for users to explore auto scale as described in: #5175

Reply:
The problem is the value seems too small for prod usage. But a larger value would break the 
tests and running the demo in small clusters. 

I would leave it off by default - users should be able to specify values in their override.

Reply:
The prob with leaving it off default is auto scaling will stop working when there is no resource/limit set.   I would prefer to set a basic value and allow user to tweak it.  If you have better suggested value than the current suggested, we could try honor it.